web/developer.actyx.com/docs/how-to/app-auth/compiling-actyx.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Deploy Actyx to production
```

Comment:
Will this document contain more sections other than compiling Actyx?
The page and a link to this page say "deploy Actyx to production", but it does not say anything about deployment.

Maybe, push the "reason of compiling your own Actyx" upwards so that the reader knows why compiling their own Actyx is relevant to production deployment